src/plane.js

```diff
@@ -0,0 +1,12 @@
+class Plane {
+    tailNumber;
+
+    constructor(tailNumber) {
+        this.tailNumber = Math.random() * 100
```

Comment:
Originally had a random number entered as a tail number, thought it was cleaver at the time but later it caused problems that were beyond my ability to solve. This was removed later.